landoapi/email.py

```diff
@@ -38,19 +36,17 @@ def make_failure_email(
     msg["To"] = recipient_email
     msg["Subject"] = f"Lando: Landing of {landing_job_identifier} failed!"
 
+    revision_status_url = ""
     if REVISION_ID_RE.match(landing_job_identifier):
         # If the landing job identifier looks like a Phab revision,
         # link to the relevant view page.
         error_details_location = f"{lando_ui_url}/{landing_job_identifier}/"
-    else:
-        # If there is no relevant URL to show the user in the email, the error
-        # details in the bottom of the email is all we have to show them for now.
-        error_details_location = "below"
+        revision_status_url = f"\nSee {error_details_location} for details.\n"
```

Comment:
nit: having a variable named `revision_status_url` would imply it contains a URL, but that is not the case here.